client/src/Questionnaires/QuestionnairesList.js

```diff
@@ -44,6 +44,9 @@ function QuestionnairesList() {
                   <Link className="btn btn-sm btn-primary me-3" to={`/questionnaires/${questionnaire.id}/edit`}>
                     Edit
                   </Link>
+                  <Link className="btn btn-sm btn-primary me-3" to={`/questionnaires/${questionnaire.id}/questions`}>
+                    Questions
+                  </Link>
```

Comment:
Matching the new route above, we add a new button with a link to look at the Questions for a given questionnaire